ChangeLog

```diff
@@ -1,3 +1,6 @@
+* 0.9.1
```

Comment:
It's bad to deprecate or drop deprecated stuff in minor versions.
Let's leave it for 0.10.